pachyderm/templates/postgresql/storageclass.yaml

```diff
@@ -0,0 +1,23 @@
+{{- /*
```

Comment:
this file was removed

Reply:
huh, wonder why that didn't show in the rebase.  Ok I'll kill it